soc/realtek/ec/rts5912/device_power.h

```diff
@@ -0,0 +1,14 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * Copyright (c) 2024 Realtek Semiconductor Corporation, SIBG-SD7
+ * Author: Lin Yu-Cheng <lin_yu_cheng@realtek.com>
+ */
+
+#ifndef __DEVICE_POWER_H
+#define __DEVICE_POWER_H
```

Comment:
Would change, as all identifiers starting with two underscores are reserved.

```suggestion
#ifndef ZEPHYR_SOC_REALTEK_RTS5912_DEVICE_POWER_H
#define ZEPHYR_SOC_REALTEK_RTS5912_DEVICE_POWER_H
```

Reply:
We changed the identifiers in `device_power.h`, thanks.